ResearchKit/Common/ORKAnswerFormat.m

```diff
@@ -891,6 +896,68 @@ - (Class)questionResultClass {
 
 @end
 
+
+#pragma mark - ORKEligibilityAnswerFormat
+
+@implementation ORKEligibilityAnswerFormat
+
+- (instancetype)init {
+    self = [self initWithPreferredAnswer:YES];
+    return self;
+}
+
+- (instancetype)initWithPreferredAnswer:(BOOL)preferredAnswer {
+    self = [super init];
+    if (self) {
+        _preferredAnswer = preferredAnswer;
+    }
+    return self;
+}
+
+- (ORKQuestionType) questionType {
+    return ORKQuestionTypeEligibility;
+}
+
+- (Class)questionResultClass {
+    return [ORKBooleanQuestionResult class];
+}
+
+- (BOOL)isAnswerValid:(id)answer {
+    return (ORKIsAnswerEmpty(answer)) ? YES : ([answer boolValue] == self.preferredAnswer);
+}
+
+- (BOOL)isEqual:(id)object {
+    BOOL isParentSame = [super isEqual:object];
+    
+    __typeof(self) castObject = object;
+    return (isParentSame &&
+            self.preferredAnswer == castObject.preferredAnswer);
+}
+
+- (NSString *)localizedInvalidValueStringWithAnswerString:(NSString *)text {
+    return ORKLocalizedString(@"INELIGIBLE_MESSAGE", nil);
```

Comment:
Shall we let the developer to decide what message to show? Just feels one INELIGIBLE_MESSAGE won't fit all scenarios.